seed.js

```diff
@@ -1,5 +1,10 @@
 const { enums: { visibility, trust } } = require('discourse-node-api');
 
+// Discourse permissions constants
```

Comment:
It'd be great to move this into the `discourse-node-api` like the visibility and trust enums or leave a TODO if there's no time right now.

Reply:
Done! :+1: